src/storage.k.md

```diff
@@ -31,6 +31,25 @@ syntax Int ::= "#UniswapV2Pair.balanceOf" "[" Int "]" [function]
 rule #UniswapV2Pair.balanceOf[A] => #hashedLocation("Solidity", 1, A)
 ```
 
+`75506153327051474587906755573858019282972751592871715030499431892688993766217` is the value of
+`#hashedLocation("Solidity", 1, 0)`. It's presence in the storage map (as a result of writes to
+`balanceOf[0]`) means that `K` cannot be sure that writes to any other storage mappings do not
+colide with this one. We assume here that this is not the case.
```

Comment:
This is cool. I bumped into it when asserting `FeeTo == 0`.

IIUC what's causing issues is the mixing of concrete and symbolic values in the memory map, right? If so, I think it could be made clearer.

Reply:
> IIUC what's causing issues is the mixing of concrete and symbolic values in the memory map, right?

This is my understanding, I'm very open to suggestions for clearer docs here.